test/org/sagebionetworks/bridge/models/schedules/ScheduleContextTest.java

```diff
@@ -81,7 +81,7 @@ public void builderWorks() {
         assertEquals(3, context.getMinimumPerSchedule());
         assertEquals("healthCode", context.getCriteriaContext().getHealthCode());
         assertEquals(TestConstants.USER_DATA_GROUPS, context.getCriteriaContext().getUserDataGroups());
-        assertEquals(now, context.getNow());
+        assertEquals(now, context.getStartsOn());
```

Comment:
Minor: This isn't a problem with the code per se, but your change did shine a spotlight on the fact that we didn't have any unit tests on withContext(). We should add that here. It's only 2 extra lines to this test.

Also, endsOn and startsOn are both set to potentially the same value. We should change the test so they use different values to make sure we aren't getting our wires crossed.